app/forms/field_handlers/number_handler.py

```diff
@@ -77,8 +79,8 @@ def _get_number_field_validators(self):
             "messages", {}
         )
 
-        for error_key, error_message in schema_validation_messages.items():
-            answer_errors[error_key] = error_message
+        for error_key in schema_validation_messages.keys():
```

Comment:
Now that `validation_messages` is a property of field handler, we can drop the `schema_validation_messages` variable and use:
```suggestion
        for error_key in self.validation_messages.keys():
```